jit-application/src/main/resources/application.yml

```diff
@@ -139,3 +139,12 @@ logging:
     org:
       hibernate:
         SQL: DEBUG
+
+camel:
+  max-redeliveries: 3
+  redelivery-delay: 10000
+  route:
+    outbox-msg: jpa:org.dcsa.jit.persistence.entity.OutboxMessage?namedQuery=poll-outbox-messages&delay=3000&transacted=true&maximumResults=10
```

Comment:
Are you sure you want transacted? As far as I know it means that if one message fails all messages in that batch fails

Reply:
This should be fine in this case as it's polling from the DB and initiating the notification flow.